src/v/security/audit/audit_log_manager.cc

```diff
@@ -103,6 +110,10 @@ class audit_sink {
     /// Returns true if _client has a value
     bool is_enabled() const { return _client != nullptr; }
 
+    double percent_full() const {
+        return is_enabled() ? _client->percent_full() : 0.0;
```

Comment:
Maybe something like `-1` would make more sense to distinguish between really 0% full and the disabled state

Reply:
Yeah good point.